x/pairing/types/expected_keepers.go

```diff
@@ -86,3 +89,13 @@ type PlanKeeper interface {
 type DowntimeKeeper interface {
 	GetDowntimeFactor(ctx sdk.Context, epochStartBlock uint64) uint64
 }
+
+type DualStakingKeeper interface {
```

Comment:
I believe we can avoid all these exports if the core functionality remains in dualstaking.